schema/data/dev/ciip_user_organisation.sql

```diff
@@ -13,22 +8,27 @@ begin;
   alter table ggircs_portal.ciip_user_organisation
     disable trigger _send_access_approved_email;
 
-  -- giving cypresstestreporter access to some organisations by default
+  -- giving cypresstestreporter && test user access to some organisations by default
   with rows as (
   insert into ggircs_portal.ciip_user_organisation(id, user_id, organisation_id, status)
   overriding system value
   values
-  (1, 3, 8, 'approved'),
-  (2, 3, 7, 'approved'),
-  (3, 6, 1, 'approved'),
-  (4, 6, 7, 'approved'),
-  (5, 6, 8, 'approved')
+  (1, 3, 2, 'approved'),
```

Comment:
Only `organisation_id` 1 is guaranteed to exist, as there is only one org when deploying the load testing data

Reply:
I'll update the load testing cron job so that it doesn't call this script

Reply:
So no changes needed here? I was looking into conditionally adding values to the ciip_user_organisation table based on the count of organisations in the organisation table with a do block here.

Reply:
I was thinking about this too. Having a different file and calling the script with a different parameter would be cleaner than having a conditional depending on a previous script's behavior

Reply:
So no, no change needed here